node/node.go

```diff
@@ -389,10 +389,12 @@ func (n *Node) ProcessBatch(ctx context.Context, header *core.BatchHeader, blobs
 }
 
 func (n *Node) ValidateBatch(ctx context.Context, header *core.BatchHeader, blobs []*core.BlobMessage) error {
+	start := time.Now()
```

Comment:
Would it be ok to take "operatorState, err := n.ChainState.GetOperatorStateByOperator(ctx, header.ReferenceBlockNumber, n.Config.ID)" out of the function. 

then in the caller context, use grafana to emit the metrics

Reply:
It's too much to have a metric to just track a RPC call, I think logging is appropriate.